unlock-app/src/unlockTypes.ts

```diff
@@ -2,6 +2,67 @@
 // throughout unlock-app.
 
 import { Card } from '@stripe/stripe-js'
+import { z } from 'zod'
+
+export const MetadataInputSchema = z.object({
+  name: z.string(),
+  defaultValue: z.string().optional(),
+  type: z.enum(['text', 'date', 'color', 'email', 'url']),
+  required: z.boolean(),
+  placeholder: z.string().optional(),
+  public: z.boolean().optional(), // optional, all non-public fields are treated as protected
+})
+
+export const PaywallCallToActionSchema = z.object({
```

Comment:
We don't support this anymore...

Reply:
Agreed. Let's remove it but keep it pass through so it doesn't fail validation.